.jenkinsci/debug-build.groovy

```diff
@@ -69,7 +69,7 @@ def doDebugBuild(coverageEnabled=false) {
     sh "cmake --build build -- -j${parallelism}"
     sh "ccache --show-stats"
     if ( coverageEnabled ) {
-      sh "lcov -i --capture --directory build --config-file .lcovrc --output-file build/reports/coverage.init.info"
+      sh "cmake --build build --target coverage.init.info"
```

Comment:
Looks as quite a strange target imo. Maybe it's better rename to `coverage-init` & `coverage-update`?

Reply:
This is the way the target was previously defined in [this commit](https://github.com/hyperledger/iroha/commit/87b60b9428778a0e07a6ba7b6060460954d0f50c). Moreover, the target defined in such a way helps to deduct report name that is used in subsequent steps